website/versioned_docs/version-5.x/recommended-setup.md

```diff
@@ -353,4 +353,4 @@ Under the rare circumstances where you do need to share UI state between single-
 
 The single-spa core team cautions against using redux, mobx, and other global state management libraries. However, if you'd like to use a state management library, we recommend keeping the state management tool specific to a single repository / microfrontend instead of a single store for all of your microfrontends. The reason is that microfrontends are not truly decoupled or framework agnostic if they all must use a global store. You cannot independently deploy a microfrontend if it relies on the global store's state to be a specific shape or have specific actions fired by other microfrontends - to do so you'd have to think really hard about whether your changes to the global store are backwards and forwards compatible with all other microfrontends. Additionally, managing global state during route transitions is hard enough without the complexity of multiple microfrontends contributing to and consuming the global state.
 
-Instead of a global store, the single-spa core team recommends using local component state for your components, or a store for each of your microfrontends. See the above section "Inter-app communication" for more related information.
+Instead of a global store, the single-spa core team recommends using local component state for your components, or a store for each of your microfrontends. See the above section "[Inter-app communication](https://single-spa.js.org/docs/recommended-setup/#inter-app-communication)" for more related information.
```

Comment:
This seems like a good change. I have one suggestion.

```suggestion
Instead of a global store, the single-spa core team recommends using local component state for your components, or a store for each of your microfrontends. See the above section "[Inter-app communication](#inter-app-communication)" for more related information.
```

Markdown doesn't require a full URL when linking to a section, just the section ID which is derived from the heading text.